output_template/python/usb_camera_demo.py

```diff
@@ -283,6 +284,61 @@ def run_sementic_segmentation(config):
         m1.show()
 
 
+def run_keypoint_detection(config):
+    global nn
+    camera_width = 320
+    camera_height = 240
+    window_name = "Keypoint Detection Demo"
+
+    input_width = config.IMAGE_SIZE[1]
+    input_height = config.IMAGE_SIZE[0]
+
+    vc = init_camera(camera_width, camera_height)
+
+    pool = Pool(processes=1, initializer=nn.init)
+    result = None
+    fps = 1.0
+
+    q_save = Queue()
+    q_show = Queue()
+
+    grabbed, camera_img = vc.read()
+    if not grabbed:
+        print("Frame is empty")
```

Comment:
What is the implication here (How bad is this situation)? 

Also, please avoid `print` and use `logging` if this is a log